### ISSUE TITLE
SearchBox on high contrast mode should show dotted border on focus

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#1903]
Currently on **high contrast mode**, the SearchBox component doesn't show a dotted border on it when it has focus. That would be nice to have for added accessibility.

**Comment by betrue-final-final** [comment]
Should have a focus indicator. That would be a solid white border

**Comment by kristoferbrown** [comment]
@betrue-final-final, is this still an issue? Currently SearchBox has a white border in HC already and focus is indicated by the icon disappearing and the text input appearing. Do we need an additional border here for focus indicating?
